extensions/oidc-client-reactive-filter/runtime/src/main/java/io/quarkus/oidc/client/reactive/filter/runtime/AbstractOidcClientRequestReactiveFilter.java

```diff
@@ -26,6 +26,10 @@ protected void initTokens() {
 
     @Override
     public void filter(ResteasyReactiveClientRequestContext requestContext) {
+        if (isClientFeatureDisabled()) {
+            LOG.debug("OIDC client feature is disabled with `quarkus.oidc-client.enabled=false`.");
```

Comment:
When exception is thrown like below, this message is clear enough to me, I know where it fail, so I can make a judgement what is not going to work and I can see why (it's disabled). But here, we don't have a stacktrace, I think few more words explaining what is happening, not just information: property is disabled